.gitignore

```diff
@@ -268,3 +268,7 @@ gallery/how_to/work_with_microtvm/micro_tvmc.py
 
 # Used in CI to communicate between Python and Jenkins
 .docker-image-names/
+/python/images/
+/python/models/
+/python/depoly/
+/python/build/
```

Comment:
These lines may not be relevant - let's get them removed

Reply:
OK